src/screens/profile/MyAccommodations/MyAccommodations.tsx

```diff
@@ -0,0 +1,84 @@
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import { useEffect } from 'react';
+import { ActivityIndicator, ScrollView } from 'react-native';
+import { useSelector } from 'react-redux';
+import { MyAccommodationListItem, Text, showAlert } from 'src/components';
+import { ScreenTemplate } from 'src/components/templates';
+import { RootStackParamList } from 'src/navigation';
+import { useAppDispatch } from 'src/store';
+import { getAccommodationLoader, getMyAccommodations } from 'src/store/selectors';
+import { AsyncThunks } from 'src/store/thunks';
+import { Accommodation } from 'src/types';
+
+import { styles } from './MyAccommodations.style';
+import { GREY_300 } from 'src/styles';
+
+const MyAccommodations = () => {
+  const dispatch = useAppDispatch();
+  const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+  const myAccommodations = useSelector(getMyAccommodations);
+  const loader = useSelector(getAccommodationLoader);
+
+  const handleEdit = (accommodation: Accommodation) => {
+    navigation.navigate('UpdateAccommodation', { accommodation });
+  };
+
+  const handleDelete = async (accommodationId: string) => {
+    showAlert('warning', {
+      message: 'Are you sure to delete this accommodation?',
+      onOkPressed: async () => {
+        const response = await dispatch(AsyncThunks.deleteAccommodation(accommodationId));
+
+        if (response.payload.success) {
+          fetchMyAccommodations();
```

Comment:
No need to make api call here, just update your state on reducers. The same for other screens